lib/measures/tbd/README.md

```diff
@@ -19,44 +19,116 @@ ModelMeasure
 ## Arguments
 
 
+### Alter OpenStudio model (Apply Measures Now)
+For EnergyPlus simulations, leave CHECKED. For iterative exploration with Apply Measures Now, UNCHECK to preserve original OpenStudio model.
+**Name:** alter_model,
+**Type:** Boolean,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
 ### Load 'tbd.json'
 Loads existing 'tbd.json' file (under '/files'), may override 'default thermal bridge' set.
 **Name:** load_tbd_json,
 **Type:** Boolean,
 **Units:** ,
-**Required:** true,
+**Required:** false,
 **Model Dependent:** false
 
 ### Default thermal bridge set
 e.g. 'poor', 'regular', 'efficient', 'code' (may be overridden by 'tbd.json' file).
 **Name:** option,
 **Type:** Choice,
 **Units:** ,
-**Required:** true,
+**Required:** false,
 **Model Dependent:** false
 
-### Alter OpenStudio model (Apply Measures Now)
-For EnergyPlus simulations, leave CHECKED. For iterative exploration with Apply Measures Now, UNCHECK to preserve original OpenStudio model.
-**Name:** alter_model,
+### Write 'tbd.out.json'
+Write out 'tbd.out.json' file e.g., to customize for subsequent runs (edit, and place under '/files' as 'tbd.json').
+**Name:** write_tbd_json,
 **Type:** Boolean,
 **Units:** ,
-**Required:** true,
+**Required:** false,
 **Model Dependent:** false
 
-### Write 'tbd.out.json'
-Write out 'tbd.out.json' file e.g., to customize for subsequent runs (edit, and place under '/files' as 'tbd.json').
-**Name:** write_tbd_json,
+### Uprate wall construction(s)
+Uprates selected wall construction(s), to meet overall Ut target
+**Name:** uprate_walls,
 **Type:** Boolean,
 **Units:** ,
-**Required:** true,
+**Required:** false,
+**Model Dependent:** false
+
+### Uprate roof construction(s)
```

Comment:
I assume the `uprate_roofs` argument doesn't do anything unless `roof_ut` is set and also that `roof_ut` doesn't do anything unless `uprate_roofs` is true.  If that is the case, you might consider removing the `uprate_roofs` argument, do not uprating if  `roof_ut` is empty, and do uprating if  `roof_ut` is not empty?

Reply:
Same for floors and walls

Reply:
Good idea. Easy fix.

Reply:
Good idea to trim down the menu items. I added a (default) "NONE" string for `roof_option` (same for wall & floors).

<img width="567" alt="none" src="https://user-images.githubusercontent.com/40636643/161992899-58bb6532-98e8-43a2-9110-26bc614ab118.png">

I can reword the item descriptions as needed, but additions to the [Guide](https://rd2.github.io/tbd/pages/basics.html) are necessary.